plone/app/locales/locales/it/LC_MESSAGES/widgets.po

```diff
@@ -465,15 +465,15 @@ msgstr "Riordina per"
 
 #: pat/preventdoublesubmit/preventdoublesubmit.js
 msgid "You already clicked the submit button. Do you really want to submit this form again?"
-msgstr ""
+msgstr "Hai già cliccato il bottone per l'invio. Vuoi realmente inviare ancora questa form?"
 
 #: pat/structure/js/actions.js
 msgid "copying"
 msgstr "Copia in corso"
 
 #: pat/relateditems/relateditems.js
 msgid "current directory:"
-msgstr ""
+msgstr "directory corrente:"
```

Comment:
Qui sono molto indeciso sull'uso di directory. Sarebbe bello usare "cartella", ma in Plone è fuorviante. Forse "posizione"?

Reply:
Concordo `posizione corrente` e' usato anche in altri posti:
```
[ale@flo locales]$ rg -i posizione.corrente locales/it
locales/it/LC_MESSAGES/widgets.po
800:msgstr "Seleziona una diversa cartella di destinazione o lascia in bianco per aggiungere i file nella posizione corrente."
locales/it/LC_MESSAGES/plone.po
11015:"rispetto alla posizione corrente."
```

Reply:
fatto